components/fxa-client/ios/FxAClient/FxAccountStorage.swift

```diff
@@ -8,13 +8,23 @@ import SwiftKeychainWrapper
 class KeyChainAccountStorage {
     internal var keychainWrapper: KeychainWrapper
     internal static var keychainKey: String = "accountJSON"
+    internal static var accessibility: KeychainItemAccessibility = .afterFirstUnlock
```

Comment:
IIUC this changes *all* FxA state to be accessible after the first unlock, including the sessionToken and sync encryption keys. Does this risk weakening our security story compared to pre-v25.0 releases of Firefox iOS?

Reply:
It shouldn't, Firefox iOS used to store (and still does) Sync and FxA state under this accessibility.